Client/Experiments/initial_experiments.json

```diff
@@ -70,6 +70,82 @@
       "proposedDuration": 28,
       "proposedEnrollment": 7,
       "referenceBranch": "control"
+    },
+    {
```

Comment:
@jeddai it's ok to keep the test experiment above?

Reply:
No, I think we're waiting for it to be ended before we merge and uplift the release experiment